### PR TITLE
docs: merge cli guide

### PR DESCRIPTION
**Pull request by yoonhyejin**
## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: metadata-ingestion/cli-ingestion.md [hunk 1]
```diff
@@ -1,66 +1,140 @@
...
+## Installing DataHub CLI
 
-```shell
-# Requires Python 3.8+
+:::note Required Python Version
+Installing DataHub CLI requires Python 3.6+.
+:::
+
+<Tabs>
+<TabItem value="pypi" label="Install from PyPI for OSS Release" default>
```
Comment (yoonhyejin) [inline metadata-ingestion/cli-ingestion.md:16]: @hsheth2 let me know if making this as tabs makes sense

Comment (hsheth2) [reply metadata-ingestion/cli-ingestion.md:16]: actually this whole gemfury thing is outdated and can be removed

File: metadata-ingestion/cli-ingestion.md [hunk 2]
```diff
@@ -1,66 +1,140 @@
...
-  type: <sink_type_name>
+  type: "datahub-rest"
   config:
-    ...
+    server: "https://<your domain name>.acryl.io/gms"
+    token: <Your API key>
 ```
+:::info Secure Your API Key
+Your API key is a signed JSON Web Token that is valid for 6 months from the date of issuance. Please keep this key secure & avoid sharing it. 
+If you are on Acryl DataHub and your key is compromised for any reason, please reach out to the Acryl team at support@acryl.io.:::
```
Comment (hsheth2) [inline metadata-ingestion/cli-ingestion.md:103]:
```suggestion
If you are on Acryl Cloud and your key is compromised for any reason, please reach out to the Acryl team at support@acryl.io.
```

File: metadata-ingestion/cli-ingestion.md [hunk 3]
```diff
@@ -1,66 +1,140 @@
...
+The final step requires invoking the DataHub CLI to ingest metadata based on your recipe configuration file.
+To do so, simply run `datahub ingest` with a pointer to your YAML recipe file:
 ```shell
 datahub ingest -c <path/to/recipe.yml>
 ```
 
+## Scheduling Ingestion
+
+Ingestion can either be run in an ad-hoc manner by a system administrator or scheduled for repeated executions. Most commonly, ingestion will be run on a daily cadence.
+To schedule your ingestion job, we recommend using a job schedule like [Apache Airflow](https://airflow.apache.org/). In cases of simpler deployments, a CRON job scheduled on an always-up machine can also work.
```
Comment (hsheth2) [inline metadata-ingestion/cli-ingestion.md:119]: we have docs on scheduling ingestion - let's link out to those

File: metadata-ingestion/cli-ingestion.md [hunk 4]
```diff
@@ -1,66 +1,140 @@
...
+Note that each source system will require a separate recipe file. This allows you to schedule ingestion from different sources independently or together.
+
+:::note Real-time Ingestion
+Real-time ingestion setup is not recommended for an initial POC as it generally takes longer to configure and is prone to inevitable system errors.
+You can find more information on real-time ingestion [here](docs/lineage/airflow.md).
```
Comment (hsheth2) [inline metadata-ingestion/cli-ingestion.md:124]: let's remove this real-time ingestion thing

we can call out that some of our sources are pushed based and don't follow this recipe pattern

File: metadata-ingestion/cli-ingestion.md [hunk 5]
```diff
@@ -1,66 +1,127 @@
...
+## Installing DataHub CLI
 
-```shell
-# Requires Python 3.8+
+:::note Required Python Version
+Installing DataHub CLI requires Python 3.6+.
+:::
+
+<Tabs>
+<TabItem value="pypi" label="Install from pip" default>
```
Comment (hsheth2) [inline metadata-ingestion/cli-ingestion.md:16]: don't need tabs here

File: metadata-ingestion/cli-ingestion.md [hunk 6]
```diff
@@ -1,66 +1,127 @@
...
-pip install 'acryl-datahub[datahub-rest]'  # install the required plugin
+pip install --upgrade acryl-datahub[mysql]
 ```
-
 Check out the [alternative installation options](../docs/cli.md#alternate-installation-options) for more reference.
 
 ## Configuring a Recipe
 
 Create a `recipe.yml` file that defines the source and sink for metadata, as shown below.
+[Recipes](metadata-ingestion/README.md#recipes) are yaml configuration files that serve as input to the Metadata Ingestion framework. Each recipe file define a single source to read from and a single destination to push the metadata.
```
Comment (hsheth2) [inline metadata-ingestion/cli-ingestion.md:48]: this line and the prior one are slightly redundant

```suggestion
[Recipes](metadata-ingestion/README.md#recipes) are yaml configuration files that serve as input to the Metadata Ingestion framework. Each recipe file defines a single source to read from and a single destination to push the metadata.
```

File: metadata-ingestion/cli-ingestion.md [hunk 7]
```diff
@@ -1,66 +1,127 @@
...
+The two most important pieces of the file are the _source_ and _sink_ configuration blocks.
+
+The _source_ configuration block defines where to extract metadata from. This can be an OLTP database system, a data warehouse, or something as simple as a file. Each source has custom configuration depending on what is required to access metadata from the source. To see configurations required for each supported source, refer to the [Sources](metadata-ingestion/README.md#sources) documentation.
+
+The _sink_ configuration block defines where to push metadata into. Each sink type requires specific configurations, the details of which are detailed in the [Sinks](metadata-ingestion/README.md#sinks) documentation.
+In Acryl DataHub deployments, you _must_ use a sink of type `datahub-rest`, which simply means that metadata will be pushed to the REST endpoints exposed by your DataHub instance. The required configurations for this sink are
```
Comment (hsheth2) [inline metadata-ingestion/cli-ingestion.md:54]:
```suggestion
In Acryl DataHub deployments, only the `datahub-rest` sink is supported, which simply means that metadata will be pushed to the REST endpoints exposed by your instance. The required configurations for this sink are
```

File: metadata-ingestion/cli-ingestion.md [hunk 8]
```diff
@@ -1,66 +1,127 @@
...
+
+1. **server**: the location of the REST API exposed by your instance of DataHub
+2. **token**: a unique API key used to authenticate requests to your instance's REST API
```
Comment (hsheth2) [inline metadata-ingestion/cli-ingestion.md:57]: let's move this down below - maybe a "using recipes with auth enabled" (don't actually like that title, but something like it)

File: metadata-ingestion/cli-ingestion.md [hunk 9]
```diff
@@ -1,66 +1,127 @@
...
 sink:
-  type: <sink_type_name>
+  type: "datahub-rest"
   config:
-    ...
+    server: "https://<your domain name>.acryl.io/gms"
+    token: <Your API key>
 ```
+:::info Secure Your API Key
+Your API key is a signed JSON Web Token that is valid for 6 months from the date of issuance. Please keep this key secure & avoid sharing it.
```
Comment (hsheth2) [inline metadata-ingestion/cli-ingestion.md:93]: the expiry time is configurable, so this is misleading

File: metadata-ingestion/cli-ingestion.md [hunk 10]
```diff
@@ -1,66 +1,119 @@
...
+## Installing Connector Plugins 
+
+Our CLI follows a plugin architecture. You must install connectors for different data sources individually. 
+For a list of all supported data sources, see [the open source docs](../docs/cli.md#sources).
+Once you've found the connectors you care about, simply install them using `pip install`.
+For example, to install the `mysql` connector, you can run
 
 ```shell
-pip install 'acryl-datahub[datahub-rest]'  # install the required plugin
+pip install --upgrade acryl-datahub[mysql]
```
Comment (hsheth2) [inline metadata-ingestion/cli-ingestion.md:34]:
```suggestion
pip install --upgrade 'acryl-datahub[mysql]'
```

**Review by hsheth2 — APPROVED** [review]

**Comment by hsheth2** [comment]
@yoonhyejin there's a merge conflict here, but this is good to merge other than that

**Pull request merged**